Sources/SwiftDocC/DocumentationService/Models/Services/Convert/ConvertRequest.swift

```diff
@@ -190,6 +196,7 @@ public struct ConvertRequest: Codable {
         includeRenderReferenceStore: Bool? = nil,
         bundleLocation: URL? = nil,
         symbolGraphs: [Data],
+        overridingDocumentationComment: [Line]? = nil,
```

Comment:
Should we have this be a `[String : [Line]]?` that maps symbol identifiers to doc comments in case there are multiple symbols in the graph?

Reply:
That's a great point. I updated the PR to reflect that.